src/browser.js

```diff
@@ -1,4 +1,13 @@
+var Client = require('./client');
+var Server = require('./server');
+var serializer = require("./transforms/serializer");
+
 module.exports = {
-  Client: require('./client'),
+  Client: Client,
+  Server: Server,
+  states: require("./states"),
+  createSerializer:serializer.createSerializer,
+  createDeserializer:serializer.createDeserializer,
+  ping: require('./ping'),
```

Comment:
Hmm indeed. I wonder how (and if) it works at all since it uses Client.connect https://github.com/PrismarineJS/node-minecraft-protocol/blob/master/src/ping.js#L45 which isn't defined in client.js
It should use that net package, so (when it's fixed) it won't work with browserify.